digits/model/images/forms.py

```diff
@@ -34,3 +30,81 @@ class ImageModelForm(ModelForm):
             tooltip = "Subtract the mean file or mean pixel for this dataset from each image."
             )
 
+    aug_flip = utils.forms.SelectField('Flipping',
+            choices = [
+                ('none', 'None'),
+                ('fliplr', 'Horizontal'),
+                ('flipud', 'Vertical'),
+                ('fliplrud', 'Horizontal and/or Vertical'),
+                ],
+            default='none',
+            tooltip = "Randomly flips each image during batch preprocessing."
+            )
+
+    aug_quad_rot = utils.forms.SelectField('Quadrilateral Rotation',
+            choices = [
+                ('none', 'None'),
+                ('rot90', '0, 90 or 270 degrees'),
+                ('rot180', '0 or 180 degrees'),
+                ('rotall', '0, 90, 180 or 270 degrees.'),
+                ],
```

Comment:
are `rot180` and `rotall` useful? Rotation by 180 degrees is the same as vertical flipping.